video/out/opengl/context_drm_egl.c

```diff
@@ -188,11 +188,15 @@ static EGLDisplay egl_get_display(struct gbm_device *gbm_device)
         PFNEGLGETPLATFORMDISPLAYEXTPROC get_platform_display = NULL;
         get_platform_display = (void *) eglGetProcAddress("eglGetPlatformDisplayEXT");
 
+#ifdef EGL_PLATFORM_GBM_MESA
         if (get_platform_display && strstr(ext, "EGL_MESA_platform_gbm"))
             return get_platform_display(EGL_PLATFORM_GBM_MESA, gbm_device, NULL);
+#endif
 
+#ifdef EGL_PLATFORM_GBM_KHR
         if (get_platform_display && strstr(ext, "EGL_KHR_platform_gbm"))
             return get_platform_display(EGL_PLATFORM_GBM_KHR, gbm_device, NULL);
+#endif
```

Comment:
Better do it like here instead: https://github.com/mpv-player/mpv/blob/master/video/out/hwdec/hwdec_vaapi_gl.c#L32

We can do that because the raw values are standardized.

Reply:
Updated the PR to use this method.